test/JDBC/expected/Test-sp_helpuser-vu-prepare.out

```diff
@@ -0,0 +1,24 @@
+CREATE PROCEDURE check_helpuser @user_or_role AS SYS.SYSNAME = NULL
```

Comment:
use filename_xxx as object names to make sure names are alway unique, applied to other places as well